Manage Logon Hour via Windows PowerShell

```diff
@@ -0,0 +1,345 @@
+
+To see and understand the challenge, let's see it first in action to understand how to build the solution. Starting by setting the Timezone to any Timezone with 0 UTC, such as London, Dublin. **+0 UTC**. Then change the **Logon Hours** GUI for a test user to only be allowed to login 1 hour as the following, and this helps in seeing how this bit is moving by the change of the timezone
+
+![Manage%20Logon%20Hour%20via%20Windows%20PowerShell%20c28909d216f44708aad42cfa6706b403/Untitled%204.png](Manage%20Logon%20Hour%20via%20Windows%20PowerShell%20c28909d216f44708aad42cfa6706b403/Untitled%204.png)
+
+Only One bit is selected
+
+Now change the Timezone to -2 or -3, such as (**UTC - 03:00) Salvador.** Open the same **Logon Hour** GUI interface again and see where the permitted login slot is now shifted.
+
+![Manage%20Logon%20Hour%20via%20Windows%20PowerShell%20c28909d216f44708aad42cfa6706b403/Untitled%205.png](Manage%20Logon%20Hour%20via%20Windows%20PowerShell%20c28909d216f44708aad42cfa6706b403/Untitled%205.png)
+
+The bit is shifted 3 bits to the left
+
+The same thing happens if the selected timezone was **(UTC +)** the bit will shift to the right with the same hour's difference between UTC the selected Timezone. Timezone bias change should be reflected in the order of the bits, so the result is always correct.
+
+In PowerShell, we can get the Timezone by using the `Get-Timezone` cmdlet. The `Get-Timezone` contains a property that shows the Timezone bias `(Get-TimeZone).baseutcoffset.hours`. Using this, it's possible to change the order of the bits based on the Timezone bais. 
+
+```powershell-console
+#Timezone in UTC -
+If ((Get-TimeZone).baseutcoffset.hours -lt 0){
+$TimeZoneOffset = $AllTheWeek.Substring(0,168+ ((Get-TimeZone).baseutcoffset.hours))
+$TimeZoneOffset1 = $AllTheWeek.SubString(168 + ((Get-TimeZone).baseutcoffset.hours))
+$FixedTimeZoneOffSet="$TimeZoneOffset1$TimeZoneOffset"
+}
+#Timezone is UTC +
+If ((Get-TimeZone).baseutcoffset.hours -gt 0){
+$TimeZoneOffset = $AllTheWeek.Substring(0,((Get-TimeZone).baseutcoffset.hours))
+$TimeZoneOffset1 = $AllTheWeek.SubString(((Get-TimeZone).baseutcoffset.hours))
+$FixedTimeZoneOffSet="$TimeZoneOffset1$TimeZoneOffset"
+}
+#Timezone is UTC 0
+if ((Get-TimeZone).baseutcoffset.hours -eq 0){
+$FixedTimeZoneOffSet=$AllTheWeek
+}
+```
+
+## Building the Final Byte array and setting the values
+
+Now the hours are aligned with the Timezone bias, and it's time to build the array and convert the string to a byte array. The result is now stored in a variable named `$FixedTimeZoneOffset`, and it should be split into a group of 8 characters be converted to Byte later.
+
+Another challenge is each 8 bits group needs to have a reverse order, so the 1's and 0's order is reflecting the correct binary number bits order, and then update the result in the `$FullByte` variable, which will be used to update the AD user information by using ~~Set-ADUser~~ cmdlet
+
+```powershell-console
```

Comment:
shold be powershell 

also remove the Parameter binding details - just leave in the parameters